src/Guild.js

```diff
@@ -25,6 +25,35 @@ class Guild {
     autoBind(this);
   }
 
+  guildRoleCreate (roleId) {
+    this.roles[this.roles.name] = roleId;
```

Comment:
`this.roles.name` is getting the value assigned to the key `name`, not the name of the new Role. You need to pass the new Role name in.

Reply:
Found the new name passed into `createRole` in the `msg.d.name` object